test/DotNetWorkerTests/AspNetCore/FunctionsHttpProxyingMiddlewareTests.cs

```diff
@@ -54,7 +54,12 @@ private static (FunctionContext FunctionContext, HttpContext HttpContext, Mock<I
                 { "test", new TestBindingMetadata("test", triggerType, BindingDirection.In ) }
             };
 
-            var functionDef = new TestFunctionDefinition(inputBindings: inputBindings);
+            var outputBindings = new Dictionary<string, BindingMetadata>()
```

Comment:
Can you clarify what you're trying to validate with this? 

Having an HTTP output binding/result wouldn't be an expected state for non-HTTP triggers.